docs/content/docs/data-structures/counter.md

```diff
@@ -12,149 +12,108 @@ menuPosition: 5
 <!-- prettier-ignore-start -->
 <!-- NOTE: This section is automatically generated by embedding the referenced file contents. Do not update these generated contents directly. -->
 
-The `SharedCounter` distributed data structure (DDS) is used to store an integer counter value that can be modified by multiple clients simultaneously.
-The data structure affords incrementing and decrementing the shared value via its `increment` method. Decrements are done by providing a negative value.
+<!-- NOTE: This section is automatically generated using @fluid-tools/markdown-magic. Do not update these generated contents directly. -->
 
-The `SharedCounter` is a specialized [Optimistic DDS][].
-It operates on communicated _deltas_ (amounts by which the shared value should be incremented or decremented), rather than direct changes to the shared value.
-In this way, it avoids the pitfalls of DDSes with simpler merge strategies, in which one user's edit may clobber another's (see [below](#why-a-specialized-dds)).
+## Using Fluid Framework libraries
```

Comment:
These contents appear to be wrong. This generated section pulls in the contents of `counter/README.md`, which should have been fixed in this PR. You may need to re-run `npm run build` under the docs directory to put these back.

Reply:
Actually, it looks like the broken READMEs (fluid-framework, counter, devtools, and devtools-core) aren't fixed yet, which is why these contents are being updated incorrectly. You'll want to make those README fixes in this PR too to fix this.

https://github.com/microsoft/FluidFramework/pull/18710#discussion_r1440857472